disdrodb/L0/readers/ARM/ARM_ld.py

```diff
@@ -192,7 +160,7 @@ def reformat_ARM_files(file_list, attrs):
         # -----------------------------------------------------------------.
         #### - Save to DISDRODB netCDF standard
         fpath = get_L1_netcdf_fpath(processed_dir, station_id)
-        write_L1_to_netcdf(ds, fpath=fpath, sensor_name=sensor_name)
+        write_L0B(ds, fpath=fpath, sensor_name=sensor_name)
```

Comment:
I think we need to update this reader. Example: `get_L1_netcdf_fpath` now is `get_L0B_fpath` if I remember correctly.

Reply:
https://github.com/EPFL-ENAC/LTE-disdrodb/blob/ba72dd5157f40ed08933ded1416496b9475d2a84/disdrodb/L0/io.py#L622

Reply:
ok @ghiggi Do you mind if I just remove this from io.py 

``` python 
def get_L0_fname(campaign_name, station_id, suffix=""):
    return get_L0A_fname(campaign_name, station_id, suffix)


def get_L0_fpath(processed_dir, station_id, suffix=""):
    return get_L0A_fpath(processed_dir, station_id, suffix)


def get_L1_netcdf_fname(campaign_name, station_id, suffix=""):
    return get_L0B_fname(campaign_name, station_id, suffix)


def get_L1_netcdf_fpath(processed_dir, station_id, suffix=""):
    return get_L0B_fpath(processed_dir, station_id, suffix)

```

And I update the reader just after ?

Reply:
ok I did it